app/views/admin/subscribers/send_email.html.erb

```diff
@@ -0,0 +1,4 @@
+<section class="container">
```

Comment:
What is this for? The API for send_email is POST request right, so i don't think you need to have this page. Instead, you might want to give another kind of notification if a user fails to send the email (e.g. because of internal error).